Tests.Unit.Presentation.Web/Models/V2/DataProcessingRegistrationResponseMapperTest.cs

```diff
@@ -244,10 +249,10 @@ public void MapDataProcessingRegistrationDTO_Maps_ExternalReferences_Properties_
             AssignExternalReferences(dpr);
 
             //Act
-            var dto = _sut.MapDataProcessingRegistrationDTO(dpr);
+            _sut.MapDataProcessingRegistrationDTO(dpr);
 
             //Assert
-            AssertExternalReferences(dpr, dto.ExternalReferences.ToList());
+            _externalReferenceResponseMapperMock.Verify(x => x.MapExternalReferenceDtoList(dpr.ExternalReferences, dpr.Reference), Times.Once);
```

Comment:
that method is not a void right?
For the reason this assertion is incomplete. We must verify that the returned mapping (whatever it is) is actually assigned to the response. I would imagine the same issue is found in the other tests